src/grammar/__snapshots__/grammar.spec.ts.snap

```diff
@@ -677,6 +677,15 @@ exports[`should fail literal-underscore-after-leading-zero.fail 1`] = `
 "
 `;
 
+exports[`should fail optional-map-value.fail 1`] = `
+"<unknown>:2:8: value cannot be optional
```

Comment:
Is it possible to point at the `Int?` here? And fix the error message to something like `map value types cannot be optional`

Reply:
Done